python/cudf/cudf/pandas/__init__.py

```diff
@@ -50,17 +52,30 @@ def install():
             mr = rmm.mr.CudaAsyncMemoryResource(initial_pool_size=free_memory)
             rmm.mr.set_current_device_resource(mr)
         elif rmm_mode == "managed":
-            mr = rmm.mr.ManagedMemoryResource()
+            mr = rmm.mr.PrefetchResourceAdaptor(rmm.mr.ManagedMemoryResource())
             rmm.mr.set_current_device_resource(mr)
         elif rmm_mode == "managed_pool":
-            mr = rmm.mr.PoolMemoryResource(
-                rmm.mr.ManagedMemoryResource(),
-                initial_pool_size=free_memory,
+            mr = rmm.mr.PrefetchResourceAdaptor(
+                rmm.mr.PoolMemoryResource(
+                    rmm.mr.ManagedMemoryResource(),
+                    initial_pool_size=free_memory,
+                )
             )
             rmm.mr.set_current_device_resource(mr)
         else:
             raise ValueError(f"Unsupported rmm mode: {rmm_mode}")
 
+        if enable_prefetching:
+            from cudf._lib import pylibcudf
```

Comment:
No real need to make this a function-local import, is there? Unless it causes some (very surprising) circular import issues I would just import it at the top level of the file unconditionally. The module will be loaded by importing cudf anyway so it doesn't save us any import time.

Reply:
Done